amazon-msk/cdk/secure-public-access/cdk.json

```diff
@@ -0,0 +1,77 @@
+{
+  "app": "npx ts-node --prefer-ts-exts bin/secure-public-access.ts",
+  "watch": {
+    "include": [
+      "**"
+    ],
+    "exclude": [
+      "README.md",
+      "cdk*.json",
+      "**/*.d.ts",
+      "**/*.js",
+      "tsconfig.json",
+      "package*.json",
+      "yarn.lock",
+      "node_modules",
+      "test"
+    ]
+  },
+  "context": {
+    "vpcId": "<your VPC ID>",
+    "mskBootstrapServers": "<Bootstrap Servers of your MSK cluster",
+    "mskClientAuthentication": "<MSK client authentication method: [mTLS, SASL/SCRAM or Unauthorized]",
+    "publicWildcardDNS": "<your public wildcard dns>",
+    "publicTlsCertificateKey": "<your public tls certificate key ARN>",
```

Comment:
Perhaps we can check to see if there is any standard naming convention to follow here?

Reply:
There is not really a standard convention that I could find. Some examples I found use camelCase some snake_case. Any preference for you?